packages/lesswrong/lib/analyticsEvents.tsx

```diff
@@ -5,8 +5,11 @@ import React, { useContext, useEffect, useState, useRef, useCallback } from 'rea
 import { hookToHoc } from './hocUtils'
 import { isClient, isServer, isDevelopment } from './executionEnvironment';
 import { ColorHash } from './vendor/colorHash';
+import { DatabasePublicSetting } from './publicSettings';
 import * as _ from 'underscore';
 
+const alwaysShowAnalyticsDebug = new DatabasePublicSetting<boolean>("alwaysShowAnalyticsDebug ", false);
```

Comment:
My interpretation of JP's request was the opposite of this (i.e. never-show rather than always show. But, would want to doublecheck with @jpaddison3 )

Reply:
Ray is correct, I would want `isDevelopment && !neverShowAnalyticsDebug`. You could imagine doing

```
type AnalyticsDebugSetting = 'never' | 'dev' | 'always'

const showAnalyticsDebugSetting = new DatabasePublicSetting<AnalyticsDebugSetting>('showAnalyticsDebug', 'dev')
```

My actual desire would be a server setting, or better yet, both, like I do here: https://github.com/LessWrong2/Lesswrong2/blob/devel/packages/lesswrong/lib/utils/logging.ts#L24

If we wanted more than two hybrid settings like that, I'd probably create a HybridSetting class, but that is definitely out of scope.

Reply:
Changed it to never/dev/always (default dev). Still a database setting for now.